src/App.tsx

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import './App.css';
 import Map from './map/Map';
+import { lightStyle } from './map/Map';
```

Comment:
The only change here is that you import multiple const from the same module on a single line;

`import { lightStyle, darkStyle } from './map/Map';`